packages/idempotency/tests/unit/IdempotencyHandler.test.ts

```diff
@@ -115,7 +109,7 @@ describe('Class IdempotencyHandler', () => {
       expect(mockResponseHook).not.toHaveBeenCalled();
     });
 
-    test('when response hook is provided, it should should call responseHook during an idempotent request', () => {
+    /* it('when response hook is provided, it should should call responseHook during an idempotent request', () => {
```

Comment:
Is this intentional or do you need to uncomment?

Reply:
Should have removed this